### PR TITLE
Add DefaultLimitMounts testcases

### PR DESCRIPTION
**Pull request by maryvictol**
The PR add testcases for default mount limits on a per-user basis #2210.

### CODE REVIEW

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_1.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- default list of mounted storages can be specified in the user-level settings
+- sensitive storages aren't available for selection in the user-level settings
+- only storages available to the user can be selected
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+
+**Preparations**:
+1. Create object 3 storages (***storage1***, ***storage2***, ***storage3***) and sensitive object storage ***storage4***
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_1.md]: Fix to "Login as the admin user from the prerequisites and create 3 object storages..."

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+| 5 | In an appeared ***Select data storages to limit mounts*** pop-up click the **Clear selection** button | |
+| 6 | Input the name of ***storage3*** created in [2210_1](2210_1.md) case Preparations into the search field | |
+| 7 | Set the checkbox in front of the row that contains ***storage3*** name | |
+| 8 | Click **OK** button | |
+| 9 | Open the **Tools** page, select the test tool | |
+| 10 | Open **Settings** tab | |
+| 11 | Expand *Execution environment* section | `All available non-sensitive storages` text is displayed in the ***Limit mounts*** field |
+| 12 | Hover over the "**Run v**" button. Click the "**Custom settings**" button in the list | |
+| 13 | Expand *Advanced* section | ***storage3*** is displayed in the **Limit mounts** field | 
+| 14 | Click the **Launch** button | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md:28]: Add step about the confirming **Launch** button click (in the pop-up) or replaced by "Launch the tool".

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+| 15 | At the **Runs** page, click the just-launched run | |
+| 16 | Expand the **Parameters** section | |
+| 17 | Click the hyperlink next to the label `CP_CAP_LIMIT_MOUNTS` | The page of the ***storage3*** is opened |
+| 18 | Repeat step 15 | | 
+| 19 | Click on *MountDataStorages* task at the left panel | *MountDataStorages* task log is displaying that contains the following text: <li> `Only 1 storages will be mounted` <li> `Found 1 available storage(s). Checking mount options.` <li> `-->{storage_name} mounted to /cloud-data/{storage_name}`, <br> where `{storage_name}` is *storage3* name |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md]: *MountDataStorages* task may not appear yet. Add step "Wait until SSH hyperlink appears", e.g.

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+| 20 | Click on "SSH" link | |
+| 21 | In the opened tab, enter and perform the command: `ls cloud-data/` | Name of the ***storage3*** is displayed |
+| 22 | Close the tab with terminal | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md]: Add block "After" where the launched run should be stopped.

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+| 10 | Click **OK** button | |
+| 11 | Click **Save** button | |
+| 12 | Login as the non-admin user from the prerequisites | |
+| 13 | Open the **Tools** page, select the test tool | |
+| 14 | Hover over the "**Run v**" button. Click the "**Custom settings**" button in the list | |
+| 15 | Expand *Advanced* section | <li> ***storage2*** is displayed in the **Limit mounts** field <li> ***storage3*** isn't displayed in the **Limit mounts** field| 
+| 16 | Click the **Launch** button | |
+| 17 | At the **Runs** page, click the just-launched run | |
+| 18 | Expand the **Parameters** section | The **Parameters** section contains `CP_CAP_LIMIT_MOUNTS: <storage2>` |
+| 19 | Click the hyperlink next to the label `CP_CAP_LIMIT_MOUNTS` | The page of the ***storage2*** is opened |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md]: Add block "After" where the launched run should be stopped.

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Tools** page, select the test tool | |
+| 3 | Open **Settings** tab | |
+| 4 | Expand *Execution environment* section | |
+| 5 | Click on the field next to **Limit mounts** label | |
+| 6 | In an appeared *Select data storages to limit mounts* pop-up click the **Clear selection** button | |
+| 7 | Input the name of ***storage1*** created in [2210_1](2210_1.md) case Preparations into the search field | |
+| 8 | Set the checkbox in front of the row that contains ***storage1*** name | |
+| 9 | Repeat steps 6-7 for ***storage2*** created in [2210_1](2210_1.md) case Preparations
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md]: Perhaps steps 7-8?
And add ` | |` to the end of the row, please.

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+| 10 | Click **OK** button | |
+| 11 | Click **Save** button | |
+| 12 | Login as the non-admin user from the prerequisites | |
+| 13 | Open the **Tools** page, select the test tool | |
+| 14 | Hover over the "**Run v**" button. Click the "**Custom settings**" button in the list | |
+| 15 | Expand *Advanced* section | <li> ***storage2*** is displayed in the **Limit mounts** field <li> ***storage3*** isn't displayed in the **Limit mounts** field| 
+| 16 | Click the **Launch** button | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md:31]: Add step about the confirming **Launch** button click (in the pop-up) or replaced by "Launch the tool".

**Review by NShaforostov — APPROVED** [review]
See minor notes in comments.

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_1.md (hunk 1 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_1.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md (hunk 2 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md:28]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md (hunk 3 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md (hunk 4 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_3.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md (hunk 7 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md:31]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md (hunk 5 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md (hunk 6 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2210/2210_4.md]: Done

**Review by NShaforostov — APPROVED** [review]

**Pull request merged**